bgpd/bgp_flowspec_util.c

```diff
@@ -581,3 +582,27 @@ int bgp_flowspec_match_rules_fill(uint8_t *nlri_content, int len,
 	}
 	return error;
 }
+
+/* return 1 if FS entry invalid or no NH IP */
+int bgp_flowspec_get_first_nh(struct bgp *bgp, struct bgp_path_info *pi,
+			      struct prefix *p)
+{
+	struct bgp_pbr_entry_main api;
+	int i;
+	struct bgp_node *rn = pi->net;
+	struct bgp_pbr_entry_action *api_action;
+
+	memset(&api, 0, sizeof(struct bgp_pbr_entry_main));
+	if (bgp_pbr_build_and_validate_entry(&rn->p, pi, &api) < 0)
+		return 1;
+	for (i = 0; i < api.action_num; i++) {
```

Comment:
Is this applicable to v6 as well?

Reply:
that depends.
I wanted to align flowspec new features with 7.0, at least.

Reply:
IMO 7.0 is only accepting bug fixes at this time.

Reply:
per call - IPv6 FS isn't supported at this time

Reply:
yes